package.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "eth-headless",
   "version": "0.1.0",
-  "private": true,
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
     "format": "prettier --write src/**/*.ts{,x}",
-    "lint": "tsc --noEmit && eslint src/**/*.ts{,x}"
+    "lint": "tsc --noEmit && eslint src/**/*.ts{,x}",
+    "clean": "shx rm -rf ./lib",
+    "build": "yarn clean && node esbuild.config.mjs && ts-patch i -s && tsc -p tsconfig.build.json",
+    "package": "shx cp -R package.json ./lib && shx cp -R README.md ./lib",
+    "publish:yalc": "yarn build && yarn package && cd lib && yalc publish --no-script --push"
   },
   "dependencies": {
     "@testing-library/jest-dom": "^5.16.4",
```

Comment:
all the testing-library and types should be in devDependancies

Reply:
Moved to devDeps ✅